From 5c012cad7858cadcaa85ec295080f3c8b21fdee0 Mon Sep 17 00:00:00 2001
From: Martin Braun <martin.braun@ettus.com>
Date: Wed, 9 Jan 2019 09:17:07 -0800
Subject: [PATCH] lib: experts: Add potentially missing but sometimes inferred
 include

This adds an include for boost/core/noncopyable.hpp. Without it, builds
would potentially fail on Boost 1.69++.
---
 host/lib/include/uhdlib/experts/expert_nodes.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/host/lib/include/uhdlib/experts/expert_nodes.hpp b/host/lib/include/uhdlib/experts/expert_nodes.hpp
index 697ca19c3..8fa183835 100644
--- host/lib/include/uhdlib/experts/expert_nodes.hpp
+++ host/lib/include/uhdlib/experts/expert_nodes.hpp
@@ -12,6 +12,7 @@
 #include <uhd/exception.hpp>
 #include <uhd/utils/dirty_tracked.hpp>
 #include <uhd/types/time_spec.hpp>
+#include <boost/core/noncopyable.hpp>
 #include <boost/function.hpp>
 #include <boost/thread/recursive_mutex.hpp>
 #include <boost/thread.hpp>

